js-exercises/sum-all/sumAll.js

```diff
@@ -0,0 +1,14 @@
+function sort(inputIntegerArray) {
```

Comment:
As a reader, I know that the function is expecting an input. Calling the input `inputIntegerArray` might be a little too descriptive. `integerArray` sounds a little better.